tests/sfbase.py

```diff
@@ -743,17 +751,17 @@ def expected_metadata():
             'WorkTypeGroupShare': incremental_last_modified,  # new
             'WorkTypeHistory': incremental_created_date,  # new
             'WorkTypeShare': incremental_last_modified,  # new
-            #'RecentlyViewed': default_full,  # new TODO verify this is not a bug  - Stream not discovered
-            #'TaskPriority': default,  # new TODO  - Stream not discovered
-            #'DeclinedEventRelation': default,  # new TODO  - Stream not discovered
-            #'AcceptedEventRelation': default,  # new TODO -  - Stream not discovered
-            #'OrderStatus': default,  # new TODO  - Stream not discovered
-            #'SolutionStatus': default,  # new TODO  - Stream not discovered
-            #'CaseStatus': default,  # new TODO  - Stream not discovered
-            #'TaskStatus': default,  # new TODO - Stream not discovered
-            # 'PartnerRole': default,  # new TODO - Stream not discovered
-            #'ContractStatus': default,  # new TODO  - Stream not discovered
-            #'UndecidedEventRelation': default,  # new TODO  - Stream not discovered
+            'RecentlyViewed': default_full,  # REST ONLY STREAM
+            'TaskPriority': default,  # REST ONLY STREAM
+            'DeclinedEventRelation': default,  # REST ONLY STREAM
+            'AcceptedEventRelation': default,  # REST ONLY STREAM
+            'OrderStatus': default,  # REST ONLY STREAM
+            'SolutionStatus': default,  # REST ONLY STREAM
+            'CaseStatus': default,  # REST ONLY STREAM
+            'TaskStatus': default,  # REST ONLY STREAM
+            'PartnerRole': default,  # REST ONLY STREAM
+            'ContractStatus': default,  # REST ONLY STREAM
+            'UndecidedEventRelation': default,  # REST ONLY STREAM
```

Comment:
Just to be more transparent.  These streams were originally commented out of sfbase but they were left in the base file and I wasn't sure why.  I later discovered the logic and dependancies.  base left them in so that a list of all keys could be created from iterating over expected_metadata.  These streams are removed from the list by other methods to account for bulk api usage.  I figured it may be more clear to label them in both places (expected_metadata and the rest only streams function) to make it more clear going forward.